modules/widgets/src/themes.ts

```diff
@@ -6,13 +6,19 @@
  * A Deck Theme is a set of CSS variables that control CSS styling of the official widgets.
  */
 export type DeckWidgetTheme = {
+  // layout
+  '--widget-margin'?: string;
```

Comment:
Why are some optional and some required? 
- Could we make all optional and use `Required<DeckTheme>` when defining a theme?
- If there are defaults, allowing themes to specify only some vars, then provide defaults for all vars?

Reply:
Yes, let's do that. There aren't defaults (they're all optional and undefined).